media-libs/mlt/mlt-6.18.0.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="An open source multimedia framework"
+DESCRIPTION="A LGPL multimedia framework designed for TV broadcasting"
+HOMEPAGE="https://www.mltframework.org/"
+COPYRIGHT="2011-2019 by Meltytech, LLC."
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://github.com/mltframework/mlt/releases/download/v$portVersion/mlt-$portVersion.tar.gz"
+CHECKSUM_SHA256="fd2a346e0cc093e5d813e04cd126c077a2d224ae4d17b5226ff7478feaa445ea"
+PATCHES="mlt-$portVersion.patchset"
+
+ARCHITECTURES="x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+libMltVersion="6"
+libMltPPVersion="3"
+libMltVersionCompat="$libMltVersion compat >= ${libMltVersion%%.*}"
+libMltPPVersionCompat="$libMltPPVersion compat >= ${libMltPPVersion%%.*}"
```

Comment:
Almost, libVersion is the same for both (=$portVersion)
libMltPPVersionCompat >= 3